alerts/templates/email/task_fail.html

```diff
@@ -30,7 +30,7 @@
         <tr><td colspan="2" height="10"></td></tr>
         <tr>
           <td width="90%">
-            <a href="{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
+            <a href="{{$.Settings.Ui.Url}}/{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
```

Comment:
This logic would work if we only used the test_logs route for viewing test logs, but we use an external logkeeper URL for server test logs. There would either need to be a conditional for logkeeper logs in the UI, or it may a good idea to append UI url in the backend itself.